python/GafferSceneUI/LightEditor.py

```diff
@@ -123,6 +123,10 @@ def __init__( self, scriptNode, **kw ) :
 
 	__columnRegistry = collections.OrderedDict()
 
+	def scene( self ) :
+
+		return self.__settingsNode["in"]
```

Comment:
This should be `return self.__plug`, so that we're returning a plug that exists in the script (and could be legitimately inspected/modified by the caller), rather than an internal plug owned by the editor.

Reply:
Fixed in 82274bb8ffdb5706aa0902f9e92fd6301439bb4f